lib/src/network/mod.rs

```diff
@@ -143,6 +144,7 @@ pub enum BackendStatus {
 
 #[derive(Debug,PartialEq,Eq)]
 pub struct Backend {
+  pub id:                 Uuid,
```

Comment:
it's just a question of cost, but I don't think it's really significant. a UUID is 128 bits, the incrementing integer could just be 32 bits. From what I see, there's one id per backend, and an option of id per client?